flang/lib/Lower/OpenMP.cpp

```diff
@@ -21,6 +23,32 @@
 
 #define TODO() llvm_unreachable("not yet implemented")
 
+static const Fortran::parser::Name *
+getDesignatorNameIfDataRef(const Fortran::parser::Designator &designator) {
+  const auto *dataRef{std::get_if<Fortran::parser::DataRef>(&designator.u)};
```

Comment:
LLVM/MLIR prefer `type var = expression;` style. For consistency with MLIR, we follow that style in lowering.